packages/cppcheck/build.sh

```diff
@@ -2,15 +2,15 @@ TERMUX_PKG_HOMEPAGE=https://github.com/danmar/cppcheck
 TERMUX_PKG_DESCRIPTION="tool for static C/C++ code analysis"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2.13.4"
+TERMUX_PKG_VERSION="2.14.0"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_METHOD=repology # Upstream only releases major versions theough GitHub. Other minor updates are released using git tags, better rely on repology for updated versiom
-TERMUX_PKG_SRCURL=https://github.com/danmar/cppcheck/archive/$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=d6ea064ebab76c6aa000795440479767d8d814dd29405918df4c1bbfcd6cb86c
+TERMUX_PKG_SRCURL=https://github.com/danmar/cppcheck/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=2d2ead75db10c5993d77c8dfe854dc42b00fac65953611bb6c83e25fc74b9d50
 TERMUX_PKG_DEPENDS="libc++"
 
 # Prevent running dmake during builds. dmake just generates Makefile which we
 # aren't using, and QT translation files, but as we are not building the GUI,
 # there is no need.  And anyways will lead to "Exec format" error as running
 # target binaries on host
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DDISABLE_DMAKE=ON"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DUSE_MATCHCOMPILER=On -DDISABLE_DMAKE=ON"
```

Comment:
Could you explain why the new option is needed?

Reply:
https://github.com/danmar/cppcheck/blob/fb15ac67930ae8d26258b4cfda07b53e8afb384d/readme.txt#L48